modules/components/src/AdvancedFacetView/FacetView.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import FacetViewNode from './FacetViewNode';
+import $ from 'jquery';
+import { debounce } from 'lodash';
+
+const serializeToDomId = path => path.split('.').join('__');
+const serializeDomIdToPath = path => path.split('.').join('__');
+
+export default class FacetView extends React.Component {
+  state = {
+    isAnimating: false,
+  };
+  componentWillReceiveProps({
+    selectedMapping,
+    path: selectedPath,
+    aggregations,
+    disPlayTreeData,
+  }) {
+    if (selectedPath) {
+      this.scrollToPath(selectedPath);
```

Comment:
This probably shouldn't be on props change, we only want to scroll on user interaction so it can go directly in the onClick handler.